.kitchen.yml

```diff
@@ -69,27 +69,32 @@ suites:
           backend: local
           controls:
           - svpc
+  - name: essential_contacts
```

Comment:
Test will also need to be added here to run in CI
https://github.com/terraform-google-modules/terraform-google-project-factory/blob/87d2df092daa2ea5efd36a98fbbd946ce5722bf7/build/int.cloudbuild.yaml#L128-L147